### DISCUSSION TITLE
Git plugin - mark up changed lines. Ideas wanted.

### DISCUSSION THREAD

**Discussion by zhabba** [orbitalquark/textadept#114] [General]
Hi. there, just discovered Textadept and it looks like it has almost everything I need. 
With one tiny exception - I'd like to have margin marks on changed lines like it's done in most of the editors or IDEs.
Would someone give me an advice what is the best approach might be? 
Run `git diff` on text changed event, parse changed lines numbers and mark it in the view?
Any ideas appreciated! Thank you.

**Comment by orbitalquark** [comment]
That is what I would do, so I think that's a good approach. Updating the view in real-time may be a bit tricky, but we can cross that bridge when you get there.

**Reply by zhabba** [reply]
Thank you for reply and inspiration! 
I'm starting then. We'll see how it goes.

**Reply by davidhealey** [reply]
How are you getting on?
